README.md

```diff
@@ -33,3 +33,18 @@ The Raspberry Pi hardware I2C does not support clock stretching. You have two op
 
 You may need to add your user to the i2c group, e.g. `sudo usermod -aG i2c nvidia`. It should just work after that.
 
+# ROS2
+## build with colcon
+`colcon build --event-handler console_direct+ --packages-select  imu_bno055`
+
+## launch
+`ros2 launch imu_bno055 imu.launch.py`
+
+## config
+In `config/default_imu_bno055.yaml` you can
```

Comment:
maybe we want to say something like:

A default configuration file listing all settable parameters is provided and is used by the provided default launch file... if you need to change any of the parameters, it is recommended that you create an appropriate configuration file for your system and reference that in an appropriate launch file for your system - the defaults provided here are for reference.